packages/db/index.js

```diff
@@ -1321,6 +1321,28 @@ export class DBClient {
     }
   }
 
+  /**
+   * @param {string} userId
+   * @param {string} customerId
+   */
+  async insertUserCustomer (userId, customerId) {
+    const { error } = await this._client
+      .from('user_customer')
+      .insert({
+        user_id: userId,
+        customer_id: customerId
+      })
+      .single()
+
+    if (error) {
+      if (error.code.startsWith('23')) {
```

Comment:
can you include a link to why this magic `23` works? what does it mean? what kind of constraintError?